pkg/loop/internal/test/logger_loop.go

```diff
@@ -51,3 +52,31 @@ func PluginLoggerTestHandshakeConfig() plugin.HandshakeConfig {
 		MagicCookieValue: "272d1867cdc8042f9405d7c1da3762ec",
 	}
 }
+
+type GRPCPluginLoggerTestPanic struct {
+	plugin.NetRPCUnsupportedPlugin
+
+	logger.Logger
+}
+
+func (g *GRPCPluginLoggerTestPanic) GRPCServer(*plugin.GRPCBroker, *grpc.Server) (err error) {
+	//Simulate panic after GRPC is started
+	go func() {
+		time.Sleep(time.Second)
+		panic("test panic")
```

Comment:
Actually, that's the only way I would expect us to be able to recover the panic at all. How does this work currently?

Reply:
Oh I see, you aren't actually "recovering" the panic. Can we tweak the title? I think others will assume catch == recover, since panic/recover is analogous to throw/catch.